modules/varbase_features/varbase_media/varbase_media.install

```diff
@@ -61,8 +61,8 @@ function varbase_media_update_8001() {
   $field_preview_view_mode_config->setData($field_preview_view_mode_config_content_data)->save();
 
   // Change name of Media embed button to be “Media Library".
-  $embed_button_config = \Drupal::service('config.factory')->getEditable('embed.button.media')
-  ->set('label','Media Library')
-  ->save();
+  $embed_button_config = \Drupal::service('config.factory')->getEditable('embed.button.media');
+  $embed_button_config->get('label')=='media' ? $embed_button_config->set('label','Media Library')
```

Comment:
- Not sure why the **shorthand!** condition at this point.
- A normal **If** statement will do with the Drupal 8 standard.
- A smarter condition could include the call for the **strtolower** php function to lower the old label then compare.